src/components/ActionBox.svelte

```diff
@@ -36,5 +36,5 @@
 </style>
 
 <div class="action-box">
-  <slot />
+  <slot class="inherit" />
```

Comment:
Is the `class="inherit"` really necessary?

Reply:
Not sure.
I use it because I was only able to get the color to change outside of the slot element.